doc/manual/en/XCSoar-developer-manual.tex

```diff
@@ -74,11 +74,59 @@ \chapter{Introduction}\label{cha:introduction}
 
 \chapter{Policy}\label{cha:policy}
 
+\section{Git Work Flow}
+
+\subsection{Version Numbering}
+
+Each release of XCSoar is denoted by a unique version number. A version
+number consists three sequential numerical fields separated a period and
+prefixed by a 'v'. The significance of the three fields are
+<major>.<minor>.<patch>. So, for instance, the version number 'v7.1.3'
+indicates XCSoar major version 7, minor version 1, and patch 3.
+
+Release versions are determined and maintained by the release manager.
+
+\subsection{Git Repository Enduring Branches}
+
+At any time the XCSoar contains two enduring branches. The principle
+enduring branch is 'master'. The lifetime of this branch is unbounded.
+The second enduring branch is the current minor version branch. The
+lifetime of this branch is the lifetime of the current minor version.
+The name of the minor version branch is in the form 'v<major>.<minor>.x'
+So, for instance, the current minor branch may be called 'v7.1.x'.
+
+With the commencement of a new minor version the release manager will
+create the required branch. The old minor branch will be retired.
+All XCSoar releases are made from the current minor branch.
+
+The master branch serves as the development branch for the next minor
+release (which may also be the first minor of the next major release).
+
+Developers should commit changes for the next minor release to the master
+branch. Bug fixes of the current minor release should be committed to
+the current minor branch, in preparation for the next patch release.
+
 \section{Writing Patches}
 
-Submit patches or \texttt{git pull} requests to the developer mailing
-list (\texttt{xcsoar-devel@lists.sourceforge.net}) or at
-\href{https://github.com/XCSoar/XCSoar}{our GitHub repository}.
+There are two methods to summit patches to the upstream XCSoar repository.
+
+The XCSoar repository is currently hosted at
+\href{https://github.com/XCSoar/XCSoar}{XCSoar GitHub repository}.
+GitHub provides a method for submitting patches using their Web interface.
+The easiest way to use this method is to make another XCSoar repository by
+ 'forkng' from 'XCSoar/XCSoar'. Make sure that your local repository
```

Comment:
typo :wink:

Reply:
But I really did mean to say "forkng". :)

Anyway I've made the change and pushed it to the repository from which I made the pull request.

For my info does this trigger some sort of notification at your end? Can you see the updated branch or do I need to start a new pull request?

Reply:
> For my info does this trigger some sort of notification at your end?

yes

> Can you see the updated branch

yes, you should be able to see f86ecab being mentioned below this comment

> do I need to start a new pull request?

no

Reply:
I cannot do anything about the German, French, Polish and Brazilian versions of the manual. How is that usually handled?

Reply:
I have no idea... but I think the developer manual is only available in English anyway, right?

Reply:
Great. This seems to be the most efficient method of submitting patches.